test-utils/state-viewer/src/main.rs

```diff
@@ -586,6 +619,46 @@ fn main() {
                 account_id
             );
         }
+        ("dump_account_storage", Some(args)) => {
+            let account_id = args.value_of("account").unwrap();
+            let storage_key = args.value_of("storage_key").unwrap();
+            let output = args.value_of("output").unwrap();
+            let block_height = args.value_of("block_height").unwrap();
+            let block_height = if block_height == "latest" {
+                LoadTrieMode::Latest
+            } else if let Ok(height) = block_height.parse::<u64>() {
+                LoadTrieMode::Height(height)
+            } else {
+                panic!("block_height shoulb be either number or \"latest\"")
+            };
+            let (runtime, state_roots, _header) =
+                load_trie_stop_at_height(store, &home_dir, &near_config, block_height);
+            for (shard_id, state_root) in state_roots.iter().enumerate() {
+                let trie = runtime.get_trie_for_shard(shard_id as u64);
+                let key = TrieKey::ContractData {
+                    account_id: account_id.to_string(),
+                    key: storage_key.as_bytes().to_vec(),
+                };
+                let item = trie.get(state_root, &key.to_vec());
+                let value = item.unwrap();
+                if let Some(value) = value {
+                    let record = StateRecord::from_raw_key_value(key.to_vec(), value).unwrap();
+                    match record {
+                        StateRecord::Data { account_id: _, data_key: _, value } => {
+                            fs::write(output, &value).unwrap();
```

Comment:
why do we only care about value but not key?

Reply:
The user who run this contract provided key: account and storage_key arg, so they already know the key